README.md

```diff
@@ -5,6 +5,26 @@ Generic encryption software
 
 ## Usage
 
+encryptio uses a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
+
+```bash
+python3 encryptio.py [-h] encryptionAlgorithm encryptionKey encryptionLibrary encryptionObjects (encryptionObjectOptions)
```

Comment:
-h soll glaub -o heißen oder?

Reply:
No, '-h' is the short option for '-help'